topydo/lib/Colors.py

```diff
@@ -58,7 +58,13 @@ def normalize_color(p_input):
         try:
             return color_names_dict[p_input]
         except KeyError:
-            return p_input
+            try:
```

Comment:
With the knowledge I gained in the master branch to optimize topydo, I think we should get rid of the `KeyError` clause, since the exception is likely to be raised when using 256 colors and hurting performance.

```python
try:
  return color_names_dict[p_input] if p_input in color_names_dict else int(p_input)
except ValueError:
  return -1
```

I don't think returning the value as-is is needed in the `ValueError` clause. It should always return an integer (xterm range and -1). The `else` would only be triggered for input values `None` or `-1`.

Unless I overlook something here..